packages/office-ui-fabric-react/src/components/Shimmer/examples/Shimmer.CustomElements.Example.tsx

```diff
@@ -11,116 +11,96 @@ const wrapperClass = mergeStyles({
     },
   },
 });
+const wrapperStyle = { display: 'flex' };
+const getCustomElementsExampleOne = (): JSX.Element => {
+  return (
+    <div style={wrapperStyle}>
+      <ShimmerElementsGroup
+        shimmerElements={[
```

Comment:
It's not great having inline arrays like this, but putting them in separate consts would decrease example readability enough that we should probably leave it as-is.